scripts/tests/test_transition_metrics.py

```diff
@@ -1,18 +1,37 @@
+import transition_metrics as tm
 import unittest
 import cymetric
-import transition_metrics as tm
 import numpy as np
 import pandas as pd
 import math
 from uuid import UUID
 from pandas._testing import assert_series_equal
 from pandas._testing import assert_frame_equal
 import sys
-sys.path.insert(1, '../')
```

Comment:
what does this do?

Reply:
That line lets python find the `transition_metrics.py` module, because it's in a different directory. This is used in other files in the repo to find analysis scripts.

Reply:
I see. You change between 1 and 0, is that significant?

Reply:
The number is the position in the list for paths to look. Because I only have the one file of that name, it looks like it doesn't make a difference if it is `0` or `1`.